docs/management/api/get-file-api.asciidoc

```diff
@@ -0,0 +1,92 @@
+[[get-file-api]]
+=== Get a file from a host
+
+Get a file from a host running {elastic-defend}.
+
+You must have the `File Operations` {kib} privilege in the Security feature as part of your role and at least an Enterprise license to perform this action.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/endpoint/action/get-file`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`endpoint_ids` |Array (String) |The IDs of endpoints where you want to issue this action. |Yes
+|`alert_ids` |Array (String) |If this action is associated with any alerts, they can be specified here. The action will be logged in any cases associated with the specified alerts. |No
+|`case_ids` |Array (String) |The IDs of cases where the action taken will be logged. |No
+|`comment` |String |Attach a comment to this action's log. The comment text will appear in associated cases. |No
+|`parameters.path` |String |The file’s full path (including the file name). |Yes
+|==============================================
+
+
+===== Example requests
+
+Get the file `/usr/my-file.txt` on a host with an `endpoint_id` value of `ed518850-681a-4d60-bb98-e22640cae2a8` and comments `Get my file`:
+
+[source,sh]
+--------------------------------------------------
+POST /api/endpoint/action/execute
+{
+  "endpoint_ids": ["ed518850-681a-4d60-bb98-e22640cae2a8"],
+  "parameters": {
+    "path": "/usr/my-file.txt",
+  },
+  "comment": "Get my file"
+}
+--------------------------------------------------
+
+
+===== Response code
+
+`200`::
+   Indicates a successful call.
+
+`403`::
+	Indicates insufficient privileges, or unsupported license level (minimum Enterprise license required).
+
+==== Response payload
+
+A JSON object with the details of the response action created.
+
+===== Example response
+
+[source,json]
+--------------------------------------------------
+{
+  "data": {
+    "id": "27ba1b42-7cc6-4e53-86ce-675c876092b2",
+      "agents": [
+          "2b292af2-5eb3-4d74-8a80-1ec6b56f66ac"
```

Comment:
Should this value match the `endpoint_id` value from the request example?
Same question for `hosts` and `agentState` values.